pages/[id].js

```diff
@@ -1,7 +1,7 @@
 import { useRouter } from "next/router";
 import ActivityDetails from "@/components/ActivityDetails";
 import { dummyData } from "@/lib/dummyData";
-import Link from "next/link";
+import { StyledBackLink } from "@/components/StyledLink";
```

Comment:
This is the import method I referred to in one of my previous comments. I would recommend to keep it consistent.